LICENSE.AGPL

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
> Would be happy to discuss this issue further (maybe offline). I think we all (?) weren't aware that the "extension message/protocol" that uses ASCII as you described it would allow new protocols, ideally, without modifying the code. I'm not sure this is possible with the current architecture.

As @bford said, there was a meeting when we discussed the RFC which led to a discussion to the wire protocol (including this above idea). I don't recall if you were here or not ( @Liamsi ). Anyway, we'll start soon to have meetings about this whole subject soon so we'll be able to expand and discuss anything related there.